[CI] XPackRestIT test {p0=sql/translate/Translate SQL} failing

**Build scan:**
https://gradle-enterprise.elastic.co/s/aapidqrkhsxqg/tests/:x-pack:qa:xpack-prefix-rest-compat:yamlRestTestV7CompatTest/org.elasticsearch.xpack.test.rest.XPackRestIT/test%20%7Bp0=sql%2Ftranslate%2FTranslate%20SQL%7D


**Reproduction line:**
```
./gradlew ':x-pack:qa:xpack-prefix-rest-compat:yamlRestTestV7CompatTest' --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=sql/translate/Translate SQL}" -Dtests.seed=C39D733199E74223 -Dtests.locale=th -Dtests.timezone=America/Guayaquil -Druntime.java=22
```

**Applicable branches:**
main

**Reproduces locally?:**
No

**Failure history:**
[Failure dashboard for `org.elasticsearch.xpack.test.rest.XPackRestIT#test {p0=sql/translate/Translate SQL}`](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Bp0%3Dsql/translate/Translate%20SQL%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.test.rest.XPackRestIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))


**Failure excerpt:**
```
org.junit.TestCouldNotBeSkippedException: Test could not be skipped due to other failures

  at org.junit.runners.model.MultipleFailureException.<init>(MultipleFailureException.java:36)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:1014)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.junit.rules.RunRules.evaluate(RunRules.java:20)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.junit.rules.RunRules.evaluate(RunRules.java:20)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at org.junit.rules.RunRules.evaluate(RunRules.java:20)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1570)

  Caused by: org.junit.AssumptionViolatedException: [sql/translate/Translate SQL] skipped, reason: [query folding changed in v 8.5, added track_total_hits: -1]

    at org.elasticsearch.test.rest.yaml.section.PrerequisiteSection.evaluate(PrerequisiteSection.java:378)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:501)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke(Method.java:580)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
    at org.junit.rules.RunRules.evaluate(RunRules.java:20)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
    at java.lang.Thread.run(Thread.java:1570)

```

I don't think this is SQL-related. The SQL test was meant to be skipped but couldn't because there were still tasks around:

```
    java.lang.AssertionError: 34 active tasks found:
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:67  -                          transport 1718281428235 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:79  -                          transport 1718281428237 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:82  -                          transport 1718281428237 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:84  -                          transport 1718281428237 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:83  -                          transport 1718281428237 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:91  -                          transport 1718281428241 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:98  -                          transport 1718281428243 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:99  -                          transport 1718281428243 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:102 -                          transport 1718281428243 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:101 -                          transport 1718281428243 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:103 -                          transport 1718281428243 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/ilm/put           rCSnMyo_QFuVLSdyPeHopQ:105 -                          transport 1718281428243 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/remove rCSnMyo_QFuVLSdyPeHopQ:106 -                          transport 1718281428245 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:107 -                          transport 1718281428247 12:23:48 38.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:118 -                          transport 1718281432032 12:23:52 34.9s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:124 -                          transport 1718281433160 12:23:53 33.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:153 -                          transport 1718281434824 12:23:54 32.1s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:160 -                          transport 1718281437291 12:23:57 29.6s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:165 -                          transport 1718281439387 12:23:59 27.5s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:171 -                          transport 1718281441217 12:24:01 25.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:176 -                          transport 1718281441998 12:24:01 24.9s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:181 -                          transport 1718281444313 12:24:04 22.6s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:184 -                          transport 1718281445453 12:24:05 21.5s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:187 -                          transport 1718281447489 12:24:07 19.4s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:190 -                          transport 1718281450250 12:24:10 16.7s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:195 -                          transport 1718281453075 12:24:13 13.8s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:201 -                          transport 1718281455577 12:24:15 11.3s   127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:204 -                          transport 1718281458434 12:24:18 8.5s    127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:207 -                          transport 1718281460082 12:24:20 6.8s    127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:210 -                          transport 1718281461479 12:24:21 5.4s    127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:213 -                          transport 1718281463447 12:24:23 3.5s    127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:216 -                          transport 1718281464958 12:24:24 1.9s    127.0.0.1 yamlRestTestV7CompatTest-0 
    cluster:admin/persistent/start  rCSnMyo_QFuVLSdyPeHopQ:219 -                          transport 1718281466336 12:24:26 621ms   127.0.0.1 yamlRestTestV7CompatTest-0 
    publish_cluster_state_update    rCSnMyo_QFuVLSdyPeHopQ:220 -                          master    1718281466338 12:24:26 618.8ms 127.0.0.1 yamlRestTestV7CompatTest-0 publication of cluster state [23]
     expected:<0> but was:<34>
```

The previous test appears to be `p0=ml/datafeeds_crud/Test get all datafeeds and stats given no datafeed exists`. The node logs before the test failure (or skip failure?) also seem to be ml-related if I'm reading this correctly.

I reassigned this to the ml team so they can maybe take a closer look.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -17,9 +17,6 @@ tests:
 - class: "org.elasticsearch.xpack.deprecation.DeprecationHttpIT"
   issue: "https://github.com/elastic/elasticsearch/issues/108628"
   method: "testDeprecatedSettingsReturnWarnings"
-- class: "org.elasticsearch.xpack.test.rest.XPackRestIT"
-  issue: "https://github.com/elastic/elasticsearch/issues/109687"
-  method: "test {p0=sql/translate/Translate SQL}"
 - class: org.elasticsearch.index.store.FsDirectoryFactoryTests
   method: testStoreDirectory
   issue: https://github.com/elastic/elasticsearch/issues/110210
@@ -95,9 +92,6 @@ tests:
 - class: org.elasticsearch.xpack.ml.integration.MlJobIT
   method: testDeleteJobAfterMissingIndex
   issue: https://github.com/elastic/elasticsearch/issues/112088
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=transform/preview_transforms/Test preview transform latest}
-  issue: https://github.com/elastic/elasticsearch/issues/112144
 - class: org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT
   issue: https://github.com/elastic/elasticsearch/issues/112147
 - class: org.elasticsearch.smoketest.WatcherYamlRestIT
@@ -141,9 +135,6 @@ tests:
 - class: org.elasticsearch.search.basic.SearchWhileRelocatingIT
   method: testSearchAndRelocateConcurrentlyRandomReplicas
   issue: https://github.com/elastic/elasticsearch/issues/112515
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=terms_enum/10_basic/Test search after on unconfigured constant keyword field}
-  issue: https://github.com/elastic/elasticsearch/issues/112624
 - class: org.elasticsearch.xpack.esql.EsqlAsyncSecurityIT
   method: testIndexPatternErrorMessageComparison_ESQL_SearchDSL
   issue: https://github.com/elastic/elasticsearch/issues/112630
@@ -230,21 +221,9 @@ tests:
 - class: org.elasticsearch.packaging.test.WindowsServiceTests
   method: test81JavaOptsInJvmOptions
   issue: https://github.com/elastic/elasticsearch/issues/113313
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=esql/50_index_patterns/disjoint_mappings}
-  issue: https://github.com/elastic/elasticsearch/issues/113315
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=wildcard/10_wildcard_basic/Query_string wildcard query}
-  issue: https://github.com/elastic/elasticsearch/issues/113316
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=mtermvectors/10_basic/Tests catching other exceptions per item}
   issue: https://github.com/elastic/elasticsearch/issues/113325
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=transform/transforms_force_delete/Test force deleting a running transform}
-  issue: https://github.com/elastic/elasticsearch/issues/113327
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=analytics/top_metrics/sort by scaled float field}
-  issue: https://github.com/elastic/elasticsearch/issues/113340
 - class: org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT
   method: test {yaml=reference/ccr/apis/follow/post-resume-follow/line_84}
   issue: https://github.com/elastic/elasticsearch/issues/113343

